test/integration/enrico.test.ts

```diff
@@ -76,4 +78,38 @@ describe('enrico', () => {
     const decrypted = bob.decrypt(pk);
     expect(bytesEqual(decrypted, plaintextBytes)).toBeTruthy();
   });
+
+  it('erico generates a message kit with conditions', async () => {
+    const label = 'fake-label';
+    const message = 'fake-message';
+    const alice = mockAlice();
+
+    const policyKey = alice.getPolicyEncryptingKeyFromLabel(label);
+
+    const ownsBufficornNFT = new Conditions.ERC721Ownership({
+      contractAddress: '0x1e988ba4692e52Bc50b375bcC8585b95c48AaD77',
+      parameters: [3591]
+    })
+
+    const conditions = new ConditionSet([ownsBufficornNFT])
+
+    const enrico = new Enrico(policyKey, undefined, conditions);
+    const encrypted = enrico.encryptMessage(toBytes(message));
+
+    const bytes = Buffer.from(encrypted.toBytes())
+    expect(bytes).toContain(188)
+
+    const conditionbytes = ConditionsIntegrator.parse(bytes).conditionsBytes
+
+    // now take the bytes and remake a ConditionSet
+    if (conditionbytes){
```

Comment:
But we DO expect `conditionbytes` to be defined, right?

Reply:
no, it will be undefined if the messagekit bytes don't contain the delimiter

Reply:
@piotr-roslaniec https://github.com/nucypher/nucypher-ts/pull/72/files#diff-033ecd338eec0994fe53c0896219baee901e20068b8fbe6259a23d5759584b06R41

Reply:
oh, sorry I see what you mean.
I was getting annoying linting issues because of the possibility of that being undefined... so it's an accommodation in the interest of removing red lines.

Reply:
Ah, I see, there is no branching in logic there. Please feel free to disable Prettier and ESLint and I will withhold any comments about style etc. until the PR is ready for review.